src/main/java/net/sf/jabref/gui/entryeditor/EntryEditor.java

```diff
@@ -305,25 +305,47 @@ private void setupFieldPanels() {
             addOtherTab(otherFields);
         }
 
+        addGeneralTab();
         // general fields from preferences
-        addGeneralTabs();
+        addAbstractReviewTabs();
         // source tab
         addSourceTab();
     }
 
-    private void addGeneralTabs() {
+    private void addAbstractReviewTabs() {
         EntryEditorTabList tabList = Globals.prefs.getEntryEditorTabList();
-        for (int i = 0; i < tabList.getTabCount(); i++) {
-            EntryEditorTab newTab = new EntryEditorTab(frame, panel, tabList.getTabFields(i), this, false,
-                    false, tabList.getTabName(i));
+        for (int i = 1; i < tabList.getTabCount(); i++) {
```

Comment:
Okay, then it would be nice if you could add a simple comment above to make it easier for others...
Jusdt like: //starting at 1 because general tab is zero and has its own method 
or whatever